libraries/utils/generics/applicationlogger.h

```diff
@@ -0,0 +1,18 @@
+#ifndef APPLICATIONLOGGER_H
+#define APPLICATIONLOGGER_H
+
+#include <QtGlobal>
+#include <QDebug>
+#include <QTime>
+#include <iostream>
+
+namespace UTILSLIB
+{
+class ApplicationLogger
```

Comment:
Use `class UTILSSHARED_EXPORT ApplicationLogger` so that this class is actually exported to the dll\shared object. In order for the compiler to find `UTILSSHARED_EXPORT` you also need to `#include "../utils_global.h"`